includes/class-wc-admin-loader.php

```diff
@@ -0,0 +1,533 @@
+			$feature = strtolower( $feature );
+			$file    = WC_ADMIN_ABSPATH . '/includes/' . $feature . '/class-wc-admin-' . $feature . '.php';
+			if ( file_exists( $file ) ) {
+				require_once( $file );
+				$feature = ucfirst( $feature );
+				self::$classes[] = 'WC_Admin_' . $feature;
+			}
+		}
+	}
+
+	/**
+	 * Registers a basic page handler for the app entry point.
+	 *
+	 * @todo The entry point for the embed needs moved to this class as well.
+	 */
+	public static function register_page_handler() {
+		add_submenu_page(
+			'woocommerce',
+			null,
+			null,
+			'manage_options',
+			'wc-admin',
+			array( 'WC_Admin_Loader', 'page_wrapper' )
+		);
+	}
+
+	/**
+	 * Registers all the neccessary scripts and styles to show the admin experience.
+	 */
+	public static function register_scripts() {
+		wp_register_script(
+			'wc-csv',
+			self::get_url( 'csv-export/index.js' ),
+			array(),
+			self::get_file_version( 'csv-export/index.js' ),
+			true
+		);
+
+		wp_register_script(
+			'wc-currency',
+			self::get_url( 'currency/index.js' ),
+			array( 'wc-number' ),
+			self::get_file_version( 'currency/index.js' ),
+			true
+		);
+
+		wp_register_script(
+			'wc-navigation',
+			self::get_url( 'navigation/index.js' ),
+			array(),
+			self::get_file_version( 'navigation/index.js' ),
+			true
+		);
+
+		wp_register_script(
+			'wc-number',
+			self::get_url( 'number/index.js' ),
+			array(),
+			self::get_file_version( 'number/index.js' ),
+			true
+		);
+
+		wp_register_script(
+			'wc-date',
+			self::get_url( 'date/index.js' ),
+			array( 'wp-date', 'wp-i18n' ),
+			self::get_file_version( 'date/index.js' ),
+			true
+		);
+
+		wp_register_script(
+			'wc-components',
+			self::get_url( 'components/index.js' ),
+			array(
+				'wp-components',
+				'wp-data',
+				'wp-element',
+				'wp-hooks',
+				'wp-i18n',
+				'wp-keycodes',
+				'wc-csv',
+				'wc-currency',
+				'wc-date',
+				'wc-navigation',
+				'wc-number',
+			),
+			self::get_file_version( 'components/index.js' ),
+			true
+		);
+
+		wp_register_style(
+			'wc-components',
+			self::get_url( 'components/style.css' ),
+			array( 'wp-edit-blocks' ),
+			self::get_file_version( 'components/style.css' )
+		);
+		wp_style_add_data( 'wc-components', 'rtl', 'replace' );
+
+		wp_register_style(
+			'wc-components-ie',
+			self::get_url( 'components/ie.css' ),
+			array( 'wp-edit-blocks' ),
+			self::get_file_version( 'components/ie.css' )
+		);
+		wp_style_add_data( 'wc-components-ie', 'rtl', 'replace' );
+
+		$entry = 'app';
+		if ( self::is_embed_page() ) {
+			$entry = 'embedded';
+		}
+
+		wp_register_script(
+			WC_ADMIN_APP,
+			self::get_url( "{$entry}/index.js" ),
+			array( 'wc-components', 'wc-navigation', 'wp-date', 'wp-html-entities', 'wp-keycodes', 'wp-i18n' ),
+			self::get_file_version( "{$entry}/index.js" ),
+			true
+		);
+
+		if ( function_exists( 'wp_set_script_translations' ) ) {
+			wp_set_script_translations( WC_ADMIN_APP, 'woocommerce-admin' );
```

Comment:
I mentioned this in #1859 (which happens to be fixed in this PR), but you'll need to add this translations call for each script that has i18n functions.

Reply:
Fixed in https://github.com/woocommerce/woocommerce-admin/pull/1863/commits/3d8ab59c461dbe2f9131d68dc40bd82280b9e219.